atom/browser/native_window_mac.mm

```diff
@@ -737,13 +737,21 @@ static bool FromV8(v8::Isolate* isolate, v8::Handle<v8::Value> val,
 }
 
 void NativeWindowMac::SetBounds(const gfx::Rect& bounds, bool animate) {
+  gfx::Size bSize = gfx::Size(bounds.width(), bounds.height());
+  bSize.SetToMax(GetMinimumSize());
+
+  gfx::Size maxSize = GetMaximumSize();
+  maxSize = gfx::Size(maxSize.width() == 0 ? bSize.width() : maxSize.width(),
```

Comment:
We couldn't use a straight up SetToMin call here as for some reason Electron's default `maximumSize` is `[0, 0]`.  This means we need these nasty ternary operatorst